examples/cassandra-3.11/start_cass_311.sh

```diff
@@ -13,12 +13,12 @@ docker-compose up -d backend-1
 
 # Bring up the 2nd C* node
 
-docker-compose up -f $COMPOSE_FILE -d backend-2
+docker-compose up -d backend-2
```

Comment:
Just wondering: Why do we have to remove the `-f` option?

Reply:
It was a bad copy paste at some point. As long as the compose file is in the same directory and named `docker-compose.yml` you don't need the `-f` flag.